components/library/libgnomeui/Makefile

```diff
@@ -13,12 +13,12 @@
 # Copyright 2014 Alexander Pyhalov.  All rights reserved.
 #
 
-include ../../make-rules/shared-macros.mk
```

Comment:
I initially put them in desktop/gnome2 but the category layout specifies library/* to go to library. What do you prefer?